docs/topics/subcriptions_and_registrations.md

```diff
@@ -0,0 +1,31 @@
+# Subscriptions and registrations
```

Comment:
I understand that a descrption of the diferences between subs and regs deserved an explicit topic, as it tends to confuse users and needs explicit clarifications as a FAQ item.

@mrutid what do you think?